tools/riscv/README_RISCV.md

```diff
@@ -0,0 +1,38 @@
+# Building RISC-V Target with Cmake #
+
+## Prerequisite: Build RISC-V Clang Toolchain and QEMU ##
+
+If you don't have prebuilt clang and riscv64 qemu, you can refer to the [script](https://github.com/sifive/prepare-riscv-toolchain-qemu/blob/main/prepare_riscv_toolchain_qemu.sh) to get the source. Copy the script to the zlib-ng root directory, and run it to download the source and build them. Modify the content according to your conditions (e.g., toolchain version).
+
+```
+./prepare_riscv_toolchain_qemu.sh
+```
+
+After running script, clang & qemu are built in `build-toolchain-qemu/riscv-clang/` & `build-toolchain-qemu/riscv-qemu/`.
+
+`build-toolchain-qemu/riscv-clang/` is your `TOOLCHAIN_PATH`.
+`build-toolchain-qemu/riscv-qemu/bin/qemu-riscv64` is your `QEMU_PATH`.
+
+## Cross-Compile for RISC-V Target ##
+
+```
+cmake -G Ninja -B ./build-riscv \
+  -D CMAKE_TOOLCHAIN_FILE=./cmake/toolchain-riscv.cmake \
+  -D CMAKE_INSTALL_PREFIX=./build-riscv/install \
+  -D TOOLCHAIN_PATH={TOOLCHAIN_PATH} \
+  -D QEMU_PATH={QEMU_PATH} \
+  .
+
+cmake --build ./build-riscv
+```
+
+Disable the option if there is no RVV support:
```

Comment:
CMake should determine if the compiler supports RVV. `WITH_RVV` should only be there to person compiling it, to be able to easily enable/disable the feature.

Reply:
Because we cannot detect rvv support at runtime, running the rvv code on a no-rvv target is a risk. Users could disable the rvv when the target don't support rvv.

We will have a better solution when the kernel updates `hwcap` or `hwprobe` for riscv.

Reply:
Perhaps at the top of this readme, below, `Building RISC-V Target with Cmake` it should indicate that there is no support for runtime detection atm.